src/diart/models.py

```diff
@@ -168,8 +206,55 @@ def __init__(self, model_info, hf_token: Union[Text, bool, None] = True):
         super().__init__(PyannoteLoader(model_info, hf_token))
 
     def forward(
-        self,
-        waveform: torch.Tensor,
-        weights: Optional[torch.Tensor] = None,
+            self,
+            waveform: torch.Tensor,
+            weights: Optional[torch.Tensor] = None,
     ) -> torch.Tensor:
         return self.model(waveform, weights=weights)
+
+
+class WeSpeakerSpeakerEmbeddingInference(LazyWeSpeakerSpeakerEmbedding):
```

Comment:
This class shouldn't exist, the logic should be taken care of by `EmbeddingModel.from_pyannote()`